src/NServiceBus.Core.Analyzer/NServiceBus.Core.Analyzer.csproj

```diff
@@ -3,9 +3,12 @@
   <PropertyGroup>
     <TargetFramework>netstandard1.1</TargetFramework>
     <PackageTargetFallback>portable-net45+win8</PackageTargetFallback>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" PrivateAssets="None" />
```

Comment:
This should be `PrivateAssets="All"`

Reply:
right, that was changed by VS without me noticing it. Good catch 👍